### PR TITLE
Make Xenobiology Old(ish) Again

### PR DESCRIPTION
**Pull request by Neerti**
Removes carbon slimes and new xenobio slimes and xenobiology systems in favor for an old-ish one, with hopefully the best of both worlds, leaning heavily towards the 'old' side.

Changes I can think of since this is so big, go read the new xenobio guide to get an idea on how much has changed.  http://ss13polaris.com/wiki/doku.php?id=game:guides:new_xenobio

Slimes/Xenobio;
- Slimes are now actually a threat again, and most functionality from carbon slimes has been added to the new simple animal slime.
- Each slime color has special abilities, to keep xenobiologists on their toes.  They can do things ranging from shooting lighting to being radioactive to lighting things on fire to exploding, and more.
- The slime taser is functional now.  Slime bation adapted to stun slimes and other lesser xeno lifeforms.
- Slime extracts are functional once more, and new abilities have been added.
- Sepia, Pyrite, Black, and Adamantite have been removed.
- Ruby, Emerald, Sapphire, and Amber slimes have been added to replace the above.
- Prometheans are now made from Sapphire slime extracts.
- Slime surgery was removed due to code quality issues, instead replaced with the Slime Processor, which works by clickdragging the slime into the processor and then clicking it.
- All things Xenobio now should have an examine panel entry, so you can play without reading the wiki if you want.  Extracts will even say what reactions are available.
- A slime securitron can now be built, by substituting the baton in a regular securitron's construction with a slimebation.  It's very useful for xenobiologists, as it will stun and discipline slimes automatically if nearby.
- Adds slimebaton and xeno taser to research borg module.  The weapons use the borg's cell, as standard.
- Slimes can obey simple commands.

Simple Animals;
- SA AI tweaked a lot to work with slimes and other future fun stuff.
- Adds more functionality to controlling an SA manually.  You can actually use your ranged attack now.
- SA AI now attacks full tile windows.

Misc;
- Mobs can have different 'chat bubbles' when talking.  Slimes/prometheans have a slimy bubble.  Communicator voices have their own bubble.  Synthetics and robots have a distinct bubble.  A few bubbles in talk.dmi are unused, pending future code.
- Moves buckling system from ``/obj`` to ``/atom/movable``, so mobs can buckle to other mobs.  This is used to allow slimes to latch onto people without some really ugly code (carbon slimes I'm looking at you).  This also sets the groundwork for riding on borgs in the future if someone ever wants to add that.
- Temperature gun is now worth using.  Has two modes, 'cold' and 'hot'.  Cold can kill slimes after a delay.
- Windows and Windoors are now repairable with a welder and some time.  Full tile windows also show cracks when damaged now.  The window crack sprites and idea are ported from /tg/.  Without this, Xenobio is way too tedious.
- Adds property for cells to self charge.  The slime cell uses this, but any cell in the future could as well by setting a var.
- Securitrons are now more talkative when fighting things.
- Adds reagent recipes for frost oil to solidify more things than just phoron.
- Adds slimebaton and xeno taser to RnD, locked behind moderate bio, weapon, and power levels but otherwise not needing exotic materials.
- Adds 'double' subtype for specific lockers, mainly the biohazard, firesuit, and bombsuit closets, to save space in the Xenobio lab.  The subtype holds twice as much equipment as usual, so two suits.
- Full tile windows are tougher, to be roughly equal to two glass panes in health value.
- Adds instance for full tile phoron glass.

Probably a lot of things I'm forgetting and not seeing on the list of changes.

### CODE REVIEW

**Comment by Neerti** [comment]
The world << stuff I left in should be gone now.

**Comment by Wickedtemp** [comment]
Will the old slime reactions be coming back? Were those considered a feature of old xenobio, or chemistry?

**Comment by Datraen** [comment]
Every time I heard someone complain about it, I offered to revert it outright.

Now I can enjoy the silence of giving even less of a fuck.

**Comment by Woodratt** [comment]
And we can enjoy a functional and fun Xenobio again. 

Thank you Neerti.

**Comment by Datraen** [comment]
I really should have ignored you guys saying that you weren't sure what direction you guys wanted to take it in and just did a wholesale revert PR, that probably would have made me a little happier.

**Comment by Datraen** [comment]
I hope everything with this turns out okay.

I wish you luck, Neerti. Thank you for releasing me from this.

Edit: Also Woodrat, I wish you well in the future of the server.

File: code/modules/clothing/suits/utility.dm [hunk 1]
```diff
@@ -78,7 +78,8 @@
 	name = "Radiation Hood"
 	icon_state = "rad"
 	desc = "A hood with radiation protective properties. Label: Made with lead, do not eat insulation"
-	flags_inv = BLOCKHAIR
+//	flags_inv = BLOCKHAIR
```
[the remark this answers is not in the document]
Comment (Neerti) [reply code/modules/clothing/suits/utility.dm]: The new hood sprites make me look very bald and it looked rather jarring to me.

File: code/game/gamemodes/technomancer/devices/disposable_teleporter.dm [hunk 2]
```diff
@@ -75,6 +75,5 @@
 			user << "<span class='notice'>You are teleported to \the [A].</span>"
 			uses--
 			if(uses <= 0)
-				user << "<span class='danger'>\The [src] has ran out of uses, and disintegrates from your hands, to prevent \
-				reverse engineering by outsiders.</span>"
+				user << "<span class='danger'>\The [src] has ran out of uses, and disintegrates from your hands</span>"
```
[the remark this answers is not in the document]
Comment (Neerti) [reply code/game/gamemodes/technomancer/devices/disposable_teleporter.dm]: Fixed.

File: code/modules/clothing/suits/utility.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (Anewbe) [reply code/modules/clothing/suits/utility.dm]: Alrighty.

**Pull request merged**